packages/squiggle-lang/src/rescript/Distributions/GenericDist.res

```diff
@@ -159,6 +119,89 @@ let toPointSet = (
   }
 }
 
+module Score = {
+  type scoreDistOrScalar = DistributionTypes.DistributionOperation.scoreDistOrScalar
+
+  type pointSet_ScoreDistOrScalar = D(PointSetTypes.pointSetDist) | S(float)
+  let argsMake = (
+    ~esti: scoreDistOrScalar,
```

Comment:
why shorten them? Just do "estimation" and "answer"

Reply:
We can discuss this here: https://github.com/quantified-uncertainty/squiggle/pull/589#discussion_r882031562